staging/src/k8s.io/legacy-cloud-providers/gce/gce_loadbalancer_internal.go

```diff
@@ -642,8 +652,10 @@ func (g *Cloud) ensureInternalInstanceGroupsDeleted(name string) error {
 
 	klog.V(2).Infof("ensureInternalInstanceGroupsDeleted(%v): attempting delete instance group in all %d zones", name, len(zones))
 	for _, z := range zones {
-		if err := g.DeleteInstanceGroup(name, z.Name); err != nil && !isNotFoundOrInUse(err) {
-			return err
+		if !g.AlphaFeatureGate.Enabled(AlphaFeatureNetLBRbs) {
```

Comment:
Move the flag check to outside of the loop so we can skip the loop if enabled.

Reply:
done